core/src/database/mysql_result.cc

```diff
@@ -126,7 +126,7 @@ std::string mysql_result::value_as_str(int idx) {
   if (_bind)
     retval = _bind->value_as_str(idx);
   else if (_row)
-    retval = _row[idx];
+    retval = (_row[idx]) ? _row[idx] : "";
```

Comment:
In case of NULL set in the column, we will also get "". Is that really what you want ?